lib/messagebird_credits_check_plugin.rb

```
@@ -0,0 +1,21 @@
+class MessagebirdCreditsCheckPlugin < Scout::Plugin
+  needs 'messagebird'
+
+  OPTIONS = <<-END
+    messagebird_api_token:
+      name: MessageBird API token
+  END
+
+  def build_report
+    return unless on_correct_util_server?
+
+    amount = MessageBird::Client.new(option(:messagebird_api_token)).balance.amount
+    report(balance: amount)
+  end
+
+  # Run this plugin on one util server, not both.
+  def on_correct_util_server?
+    File.exist?('/var/www/production.rom.roqua.nl/current/config/cron_state') &&
+      File.read('/var/www/production.rom.roqua.nl/current/config/cron_state').strip == 'on'
```

Kan dit niet beter met een envvar? Afhangen van installatie in een specifieke map lijkt me fragiel, zeker gecombineerd met silent failure.